parametric/test_headers_b3multi.py

```diff
@@ -53,7 +52,10 @@ def test_headers_b3multi_extract_valid(test_agent, test_library):
 
 
 @enable_b3multi()
-@pytest.mark.skip_library("golang", "not implemented")
+@pytest.mark.skip_library(
+    "golang",
+    "BUG: the parent_id will be 0 in any span that didn't have a parent context ; if extract fails - the parent context is nil",
```

Comment:
I ran into this exact same issue in the b3 tests. I believe the parent_id being 0 is the correct behavior and perhaps the test should change?
I would recommend:
```
assert span.get("span_id") != 0
assert span.get("parent_id") == 0
```

Reply:
The context extracted with this propagator will be later used in `tracer.ChildOf(ctx)` option, so the `assert span.get("span_id") != 0` is technically incorrect. As spanID of the context IS the parentID of the span in question. 
Maybe it's possible to introduce an exception for Go though